
Randaugment!
Number of final features: 1024
Number of model parameters: 68150983
Traceback (most recent call last):
  File "/home/yu-jw19/venom/project2/train.py", line 579, in <module>
    main()
  File "/home/yu-jw19/venom/project2/train.py", line 406, in main
    train_metrics = train(train_loader, model, ce_criterion, optimizer, epoch)
  File "/home/yu-jw19/venom/project2/train.py", line 444, in train
    output = model(x)
  File "/home/yu-jw19/.local/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/yu-jw19/venom/project2/networks/resnext.py", line 106, in forward
    x = self.stage_2(x)
  File "/home/yu-jw19/.local/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/yu-jw19/.local/lib/python3.9/site-packages/torch/nn/modules/container.py", line 139, in forward
    input = module(input)
  File "/home/yu-jw19/.local/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/yu-jw19/venom/project2/networks/resnext.py", line 35, in forward
    bottleneck = F.relu(self.bn(bottleneck), inplace=True)
  File "/home/yu-jw19/.local/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/yu-jw19/.local/lib/python3.9/site-packages/torch/nn/modules/batchnorm.py", line 167, in forward
    return F.batch_norm(
  File "/home/yu-jw19/.local/lib/python3.9/site-packages/torch/nn/functional.py", line 2281, in batch_norm
    return torch.batch_norm(
RuntimeError: CUDA out of memory. Tried to allocate 256.00 MiB (GPU 0; 23.70 GiB total capacity; 10.63 GiB already allocated; 37.56 MiB free; 11.26 GiB reserved in total by PyTorch)
Traceback (most recent call last):
  File "/home/yu-jw19/venom/project2/train.py", line 579, in <module>
    main()
  File "/home/yu-jw19/venom/project2/train.py", line 406, in main
    train_metrics = train(train_loader, model, ce_criterion, optimizer, epoch)
  File "/home/yu-jw19/venom/project2/train.py", line 444, in train
    output = model(x)
  File "/home/yu-jw19/.local/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/yu-jw19/venom/project2/networks/resnext.py", line 106, in forward
    x = self.stage_2(x)
  File "/home/yu-jw19/.local/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/yu-jw19/.local/lib/python3.9/site-packages/torch/nn/modules/container.py", line 139, in forward
    input = module(input)
  File "/home/yu-jw19/.local/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/yu-jw19/venom/project2/networks/resnext.py", line 35, in forward
    bottleneck = F.relu(self.bn(bottleneck), inplace=True)
  File "/home/yu-jw19/.local/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/yu-jw19/.local/lib/python3.9/site-packages/torch/nn/modules/batchnorm.py", line 167, in forward
    return F.batch_norm(
  File "/home/yu-jw19/.local/lib/python3.9/site-packages/torch/nn/functional.py", line 2281, in batch_norm
    return torch.batch_norm(
RuntimeError: CUDA out of memory. Tried to allocate 256.00 MiB (GPU 0; 23.70 GiB total capacity; 10.63 GiB already allocated; 37.56 MiB free; 11.26 GiB reserved in total by PyTorch)